pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go

```diff
@@ -87,6 +92,7 @@ func NewUpgradeBackupController(
 		clusterVersionLister:  clusterVersionInformer.Lister(),
 		targetImagePullSpec:   targetImagePullSpec,
 		operatorImagePullSpec: operatorImagePullSpec,
+		featureGateAccessor:   featureGateAccessor,
```

Comment:
Looks good, I'm guessing this will give us access to reading the EtcdBackupAPI feature gate e.g:
```Go
featureGates, err := featureGateAccessor.CurrentFeatureGates()
if err != nil {
	return nil, err
}
etcdBackupAPIEnabled := featureGates.Enabled(configv1.FeatureGateEtcdBackupAPI)
```